templates/repo/diff/compare.tmpl

```diff
@@ -12,11 +12,32 @@
 				{{.i18n.Tr "action.compare_commits_general"}}
 			{{ end }}
 		</h2>
+		{{ $BaseCompareName := $.BaseName -}}
```

Comment:
```suggestion
		{{- $BaseCompareName := $.BaseName -}}
```
??

Reply:
The current pr as written keeps the spacing of the outputted file exactly the same as it was before.

Adding that would coalesce the two tags together.